src/Microsoft.AspNetCore.Mvc.ViewFeatures/project.json

```diff
@@ -47,18 +47,15 @@
       "type": "build"
     },
     "Microsoft.Extensions.WebEncoders": "1.0.0-*",
-    "Newtonsoft.Json": "8.0.3",
+    "Newtonsoft.Json": "8.0.4-beta1",
```

Comment:
@pranavkm correct, NuGet.exe will fail on this. dotnet pack allows this I believe, but that will soon be replaced by nuget.exe's pack. I suggest waiting for a stable version of newtonsoft.json before updating.